packaging/docker/kubernetes/test_config.yaml

```diff
@@ -0,0 +1,285 @@
+        {"value": "--class"},
+        {"value": "storage"},
+        {"value": "--locality_zoneid"},
+        {"type": "Environment", "source": "FDB_ZONE_ID"},
+        {"value": "--locality_instance-id"},
+        {"type": "Environment", "source": "FDB_INSTANCE_ID"},
+        {"value": "--locality_process-id"},
+        {"type": "Concatenate", "values": [
+          {"type": "Environment", "source": "FDB_INSTANCE_ID"},
+          {"value": "-"},
+          {"type": "ProcessNumber"}
+        ]},
+        {"value": "--logdir"},
+        {"value": "/var/fdb/logs"},
+        {"value": "--trace_format"},
+        {"value": "json"}
+      ]
+    }
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fdb-kubernetes-example
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: fdb-kubernetes-example
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - "pods"
+    verbs:
+      - "get"
+      - "watch"
+      - "update"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: fdb-kubernetes-example
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: fdb-kubernetes-example
+subjects:
+- kind: ServiceAccount
+  name: fdb-kubernetes-example
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fdb-kubernetes-example-client
+spec:
+  replicas: 2
+  revisionHistoryLimit: 1
+  selector:
+    matchLabels:
+      app: fdb-kubernetes-example-client
+  template:
+    metadata:
+      labels:
+        app: fdb-kubernetes-example-client
+      name: fdb-kubernetes-example-client
+    spec:
+      volumes:
+        - name: config-map
+          configMap:
+            name: fdb-kubernetes-example-config
+            items:
+              - key: fdb.cluster
+                path: fdb.cluster
+        - name: dynamic-conf
+          emptyDir: {}
+      initContainers:
+        - name: foundationdb-kubernetes-init
+          image: foundationdb/foundationdb-kubernetes:latest
+          imagePullPolicy: IfNotPresent
+          args:
+            - "--mode"
+            - "init"
+            - "--input-dir"
+            - "/var/input-files"
+            - "--output-dir"
+            - "/var/output-files"
+            - "--copy-file"
+            - "fdb.cluster"
+            - "--require-not-empty"
+            - "fdb.cluster"
+            - "--copy-library"
+            - "6.1"
+            - "--copy-library"
+            - "6.2"
+            - "--copy-primary-library"
+            - "6.3"
+            - "--copy-binary"
+            - "fdbcli"
+          volumeMounts:
+            - name: config-map
+              mountPath: /var/input-files
+            - name: dynamic-conf
+              mountPath: /var/output-files
+      containers:
+        - name: client
+          image: foundationdb/foundationdb-sample-python-app:latest
+          imagePullPolicy: Always
+          env:
+            - name: FDB_CLUSTER_FILE
+              value: /var/dynamic-conf/fdb.cluster
```

Comment:
I've often wondered if this could be hardcoded, or at least set to this path as a default.  If we had a strong opinion about where these files should live, then we could eliminate a lot of downstream configuration.

(Of course, we'd need a naming convention for multi-cluster setups:  `fdb*.cluster`?)